langkit/expressions/base.py

```diff
@@ -2905,6 +2905,8 @@ def construct_static(elements, array_type, abstract_expr=None):
             )
```

Comment:
Can you add to the commit message the reason why you need this? I guess something else breaks without this patch, but it would be useful to know what